### PR TITLE
API to access DB

### LINKED ISSUE #58: Access to the private log DB
Hello guys, 
Thank you for this amazing tool! Is it possible to **access/download the whole public flight log database** (instead of writing a scraper that pulls it)? This would make it easier to experiment with all the timeseries available so far and feed them to some LSTMs etc.

### PR DESCRIPTION
**Pull request by joaeba**
- this closes #58 
- add API (using a tornado request handler) to get access to the public DB via entry point review.px4.io/dbinfo
- request returns a JSON file with a list of database entries: log id + other information
- add test_db_info_api.py to test the API by downloading a log file

### CODE REVIEW

File: tornado_handlers.py [hunk 1]
```diff
@@ -671,7 +672,71 @@ def get(self, *args, **kwargs):
         template = _ENV.get_template(BROWSE_TEMPLATE)
         self.write(template.render(table_data=table_header + table_data + table_footer))
 
+class DBInfoHandler(tornado.web.RequestHandler):
+    """ Get database info Tornado request handler """
 
+    def get(self, *args, **kwargs):
```
Comment (bkueng) [inline tornado_handlers.py]: At some point we should merge this into a common base class shared with the `BrowseHandler` to minimize code duplication, but for now it's ok.

File: test_db_info_api.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import requests
```
Comment (bkueng) [inline test_db_info_api.py]: Are you planning on reworking this into a full download script? For testing, this is fine, but we should include a script that handles the full download, with specifying path, avoid redownloading, etc.

Comment (joaeba) [reply test_db_info_api.py]: good point, I need a download script anyways.

File: download_logs.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+!import os, glob
```
Comment (anassinator) [inline download_logs.py]: @joaeba sorry just lurking here, but noticed this typo was committed by accident instead of the usual shebang

**Comment by bkueng** [comment]
Rebased & applied. And I fixed the script: https://github.com/PX4/flight_review/commit/affde9c72fb7523fc192a91581977287f37c4b94

**Pull request closed without merging**